spec/models/user_spec.rb

```diff
@@ -12,22 +12,24 @@
   end
 
   it 'should connect to mailchimp if mailing list and confirmed' do
-    VCR.use_cassette('models/user_spec/mailing_list') do
-      # These tests are pretty meaningless with the VCR, but we also don't
-      # want to add fake sign-ups every time tests are run.
-      user = FactoryGirl.create(:confirmed_user)
-      user.mailing_list = true
-      user.save
-    end
+    stub_request(:post, "https://us8.api.mailchimp.com/2.0/lists/list")
+      .to_return(:body => '{"total":1,"data":[{"id":"7ec999ba49","web_id":540717,"name":"OpenFarm Helpers","date_created":"2015-04-03 08:25:15","email_type_option":false,"use_awesomebar":true,"default_from_name":"The OpenFarm Team","default_from_email":"kevin@openfarm.cc","default_subject":"","default_language":"en","list_rating":0,"subscribe_url_short":"http:\/\/eepurl.com\/biTabv","subscribe_url_long":"http:\/\/openfarm.us8.list-manage.com\/subscribe?u=&id=7ec999ba49","beamer_address":"us8@inbound.mailchimp.com","visibility":"pub","stats":{"member_count":24,"unsubscribe_count":0,"cleaned_count":0,"member_count_since_send":27,"unsubscribe_count_since_send":0,"cleaned_count_since_send":0,"campaign_count":0,"grouping_count":0,"group_count":0,"merge_var_count":1,"avg_sub_rate":0,"avg_unsub_rate":0,"target_sub_rate":0,"open_rate":0,"click_rate":0,"date_last_campaign":null},"modules":[]}],"errors":[]}')
+    stub_request(:post, "https://us8.api.mailchimp.com/2.0/lists/subscribe")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.